cost/turbonomics/credential_refresh/turbonomic_cred_refresh.pt

```diff
@@ -0,0 +1,166 @@
+name "Turbonomic Credential Refresh"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that refreshes the cookie used to authenticate with Turbonomic APIs"
+severity "low"
+category "Operational"
+default_frequency "hourly"
+info(
+  version: "0.1",
+  provider: "",
+  service: "",
+  policy_set: "",
+	publish: "false"
+)
+
+parameter "turbo_username" do
+  type "string"
+  label "Turbonomic Username"
+  description "The username used to authenticate with Turbonomic APIs"
+end
+
+parameter "turbo_password" do
+  type "string"
+  no_echo true
+  label "Turbonomic Password"
+  description "The password used to authenticate with Turbonomic APIs"
+end
+
+parameter "turbo_host" do
+  type "string"
+  label "Turbonomic Host"
+  description "Your Turbonomic host or IP."
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+credentials "auth_flexera" do
+  schemes "oauth2"
+  label "flexera"
+  description "Select Flexera One OAuth2 credentials"
+  tags "provider=flexera"
+end
+
+###############################################################################
+# Datasources & Scripts
+###############################################################################
+
+# Retrieves all applied policies in the project where this policy is applied
+datasource "applied_policies" do
+  request do
+    auth $auth_flexera
+    host rs_governance_host
+    path join(["/api/governance/projects/", rs_project_id, "/applied_policies"], "")
+    header "Api-Version", "1.0"
+  end
+  result do
+    encoding "json"
+    collect jq(response, ".items[]") do
+      field "id", val(col_item, "id")
+      field "name", val(col_item, "name")
+      field "info_source", jq(col_item, ".info.source")
+    end
+  end
+end
+
+# Filters the applied policy where the info.source is set to either "Turbonomic" or "Turbonomics"
+datasource "turbo_applied_policies" do
```

Comment:
For all data sources I think we can start using this docs to do better practices and conventions: https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices#Block-Naming-Conventions.